nvdaHelper/localWin10/sconscript

```diff
@@ -65,14 +37,13 @@ localWin10Lib = env.SharedLibrary(
 	],
 	LIBS=["oleaut32", localLib[2]],
 )
-if signExec:
-	env.AddPostAction(localWin10Lib[0], [signExec])
-env.Install(libInstallDir, localWin10Lib)
 
 # UWP dlls can only be dynamically linked with the CRT,
 # but some systems might not have this version of the CRT.
 # Therefore, we must include it.
-vcRedist = os.path.join(progFilesX86, r"Microsoft Visual Studio 14.0\VC\redist\onecore\x86\Microsoft.VC140.CRT")
+vcRedist = os.path.join(progFilesX86, r"Microsoft Visual Studio\2017\Community\VC\Redist\MSVC\14.11.25415\onecore\x86\Microsoft.VC141.CRT")

```

Comment:
Would it be possible to get rid of the 14.11.25415 part here by using glob and only use the most recent version available? One problem I see with the current approach is that Appveyor silently updates its Visual Studio 2017 image to a newer build of VS, which might bundle a newer version of the redistributable. Strange enough, when I updated to Visual Studio 2017.4, i lost 14.11.25415 and have now 14.11.25325.